urlUpdater/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file and some test resources are still in the `urlUpdate` folder that was renamed to `url-updater`.
Can you quickly fix this.
IMHO we should also already add this project as `module` to the parent `pom.xml` so that the CI build (esp. for PRs) will fail if this code here does not compile or tests fail.
Can you do that as well?

Reply:
> IMHO we should also already add this project as `module` to the parent `pom.xml` so that the CI build (esp. for PRs) will fail if this code here does not compile or tests fail. Can you do that as well?

@hohwille does [this commit ](https://github.com/devonfw/ide/pull/1000/commits/b4503e9cd1563a8a6a95eee365508596408d4133) address what you meant properly (as I'm rather new to maven)?

Reply:
Fine. All aspects of this comment are addressed so I can resolve this. Thanks 👍